test/parallel/test-process-versions.js

```diff
@@ -36,7 +36,7 @@ const actual_keys = Object.keys(process.versions).sort();
 
 assert.deepStrictEqual(actual_keys, expected_keys);
 
-const commonTemplate = /^\d+\.\d+\.\d+(?:-.*)?$/;
+const commonTemplate = /^\d+\.\d+\.\d+(?:\.\d+)?(?:-.*)?$/;
```

Comment:
It's because the `zlib` upstream now contains the codename and the patch version:

```
rafaelgss@rafaelgss-desktop:~/repos/os/node$ ./node test/parallel/test-process-versions.js
node:assert:1027
    throw err;
    ^

AssertionError [ERR_ASSERTION]: The input did not match the regular expression /^\d+\.\d+\.\d+(?:-.*)?$/. Input:

'1.2.12.1-motley'

    at Object.<anonymous> (/home/rafaelgss/repos/os/node/test/parallel/test-process-versions.js:46:8)
    at Module._compile (node:internal/modules/cjs/loader:1175:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1229:10)
    at Module.load (node:internal/modules/cjs/loader:1038:32)
    at Module._load (node:internal/modules/cjs/loader:879:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:82:12)
    at node:internal/main/run_main_module:23:47 {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: '1.2.12.1-motley',
  expected: /^\d+\.\d+\.\d+(?:-.*)?$/,
  operator: 'match'
}
```